<launch>
	<node pkg="serial" type="serial_node"  name="serial_node">
		<param name="serial_port" value="/dev/ttySAC1"/>
		<param name="baud_rate" value="1000000"/>
	</node>
</launch>
